src/mixins/document-mixin.ts

```diff
@@ -192,4 +202,52 @@ export default class DocumentMixin extends Vue {
     if (sizeKB > 1) return `${sizeKB.toFixed(0)} KB`
     return `${size} bytes`
   }
+
+  /**
+   * Uploads the specified file to Document Record Service.
+   * @param file the file to upload
+   * @param documentClass the document class defined for the document service. e.g. 'CORP'
+   * @param documentType the type of document. e.g. 'CNTA'
+   * @param consumerDocumentId the identifier of one or more documents associated with the filing.
```

Comment:
How do you specify "more than one" document?

Reply:
So, there is another unique ID(DocumentServiceID) for each document.
This consumerDocumentId is used to group of documents for the same application.